cartridge-cli.lua

```diff
@@ -1538,6 +1538,15 @@ local HEADERIMMUTABLE=63
 -- There are way more tags in the spec than what I've included here
 -- both for signature header and regular header. Most of them are
 -- optional.
+--
+-- Explanation and values for most of this tags can be found in documentation:
+-- - http://ftp.rpm.org/max-rpm/s1-rpm-file-format-rpm-file-format.html
+-- - https://docs.fedoraproject.org/ro/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch-package-structure.html
+--
+-- But I didn't find documentation for some tags (PAYLOADDIGEST, PAYLOADDIGESTALGO),
+-- so, I got these values from the rpm repo:
+-- - https://github.com/rpm-software-management/rpm/blob/master/lib/rpmtag.h
```

Comment:
And may be this PR? https://github.com/rpm-software-management/rpm/issues/163

Reply:
I propose not to. It's not mandatory, and it will complicate packing.